activesupport/lib/active_support/core_ext/module/delegation.rb

```diff
@@ -6,11 +6,12 @@ class Module
   # option is not used.
   class DelegationError < NoMethodError; end
 
+  RUBY_RESERVED_KEYWORDS = %w(alias and BEGIN begin break case class def defined? do
```

Comment:
Not sure the constant naming and defined location are good. 

@matthewd What do you think?